docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -58,29 +58,29 @@ components:
       command: ['sleep', 'infinity']
       env:
         - name: GOPATH
-          value: $(CHE_PROJECTS_ROOT)/go
+          value: $(PROJECTS_ROOT)/go
         - name: GOCACHE
           value: /tmp/go-cache
 commands:
   - id: compile and run
     exec:
       component: go-cli
       commandLine: “go get -d && go run main.go”
-      workingDir: “${CHE_PROJECTS_ROOT}/src/github.com/acme/my-go-project”
+      workingDir: “${PROJECTS_ROOT}/src/github.com/acme/my-go-project”
```

Comment:
Use ASCII characters only. There quotation marks were typographical, which I'm quite sure would cause an error.

```suggestion
      workingDir: "${PROJECTS_ROOT}/src/github.com/acme/my-go-project"
```